website/core/Footer.js

```diff
@@ -102,7 +102,7 @@ class Footer extends React.Component {
 
             <div className="linux-foundation-logo">
               <a href="https://www.linuxfoundation.org/projects/">
-                <img src="/img/LF-logo.png" />
+                <img src="img/LF-logo.png" alt="The Linux Foundation Projects"/>
```

Comment:
This change will break the LF logo again. Please use the `/`. See #92

Reply:
Sorry. I didn't see this. edited the commit. please check.